x-pack/plugins/observability/public/pages/alerts/alerts_status_filter.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonGroup, EuiButtonGroupOptionProps } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { AlertStatus, AlertStatusFilter } from '../../../common/typings';
+
+export interface AlertStatusFilterProps {
+  status: AlertStatus;
+  onChange: (id: string, value: string) => void;
+}
+
+export const allAlerts: AlertStatusFilter = {
+  status: AlertStatus.All,
+  query: '',
+  label: i18n.translate('xpack.observability.alerts.alertStatusFilter.showAll', {
+    defaultMessage: 'Show all',
+  }),
+};
+
+export const activeAlerts: AlertStatusFilter = {
+  status: AlertStatus.Active,
+  query: `kibana.alert.status : "active"`,
```

Comment:
@fkanout same comment here. Let's not hardcode field name and value and let's use these:

import { ALERT_STATUS_ACTIVE } from '@kbn/rule-data-utils/alerts_as_data_status';
import { ALERT_STATUS } from '@kbn/rule-data-utils/technical_field_names';

Reply:
Updated accordingly